website/src/pages/index.module.css

```diff
@@ -9,23 +9,30 @@
   overflow: hidden;
   background-image: linear-gradient(to bottom, rgba(0, 0, 0, 0.3) 0%, rgba(0, 0, 0, 0.3) 100%),
     url("../../static/img/hero.svg");
-  background-position: center;
+  background-position: 0% 75%;
```

Comment:
Have you checked this change for all the screen resolutions?

Reply:
I checked it. Making this change since the design team decide to use the lower part of the image